angular-ngrx-scss/src/app/state/profile/profile.effects.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
This would be the best file to keep and add actual tests to, to double check our side effects.

Reply:
So do I need to increase coverage here and check that HTTP requests are being fired?

Reply:
Doesn't have to be complete coverage (and I think the getUserInfo call is already handled in it's own test file), but something to show what a test would look like I think is what we'd want. Maybe one to make sure the calls are firing like we expect and that we get some mocked results back? (Can also make a backlog ticket for this, I don't think this is super pressing)